example/src/main/scala/com/thenewmotion/ocpi/example/ExampleApp.scala

```diff
@@ -17,38 +18,25 @@ class ExampleHandshakeService(implicit http: HttpExt) extends HandshakeService(
   ourLogo = None,
   ourWebsite = None,
   ourBaseUrl = Uri("www.ocpi-example.com"),
-  ourPartyId = "exp",
-  ourCountryCode = "NL"
+  ourPartyId = PartyId("exp"),
+  ourCountryCode = CountryCode("NL")
 ) {
-  override protected def persistHandshakeReactResult(
-    version: Version,
-    existingTokenToConnectToUs: String,
-    newTokenToConnectToUs: String,
-    credsToConnectToThem: Creds,
-    endpoints: Iterable[Endpoint]) = ???
 
-  override protected def persistUpdateCredsResult(
-    version: Version,
-    existingTokenToConnectToUs: String,
-    newTokenToConnectToUs: String,
-    credsToConnectToThem: Creds,
-    endpoints: Iterable[Endpoint]) = ???
+  override protected def persistPartyPendingRegistration(partyName: String, globalPartyId: GlobalPartyId,
+    newTokenToConnectToUs: TheirToken) = ???
 
-  override protected def persistHandshakeInitResult(
-    version: Version,
-    newTokenToConnectToUs: String,
-    newCredToConnectToThem: Creds,
-    endpoints: Iterable[Endpoint]) = ???
+  override protected def removePartyPendingRegistration(globalPartyId: GlobalPartyId) = ???
 
-  override protected def persistPartyPendingRegistration(
-    partyName: String,
-    countryCode: String,
-    partyId: String,
-    newTokenToConnectToUs: String) = ???
+  override def credsToConnectToUs(globalPartyId: GlobalPartyId) = ???
 
-  override protected def removePartyPendingRegistration(tokenToConnectToUs: String) = ???
+  override protected def persistHandshakeReactResult(version: VersionNumber, globalPartyId: GlobalPartyId,
```

Comment:
Pretty sure Dmitrii will change this back to how it was if he's ever to work on it...
I'd love to have a common style guide for the MSP team that we could also put in our IDEs to prevent such formatting ping-pong